test/e2e/json-rpc/switchEthereumChain.spec.js

```diff
@@ -29,6 +29,28 @@ describe('Switch Ethereum Chain for two dapps', function () {
       async ({ driver }) => {
```

Comment:
we should possibly add either a comment or some note in the description about how this toggles off the per dapp selected network setting